docs/product-getting-started-guide/src/main/asciidoc/executing-proc.adoc

```diff
@@ -1,20 +1,43 @@
 
 .Prerequisites
 
-The {KIE_SERVER} is deployed and connected to the {PRODUCT}.
+ * The {KIE_SERVER} is deployed and connected to the {PRODUCT}
+ * You have deployed the *Mortgage Process* application
+ * The user or users working on the tasks must be members of the following groups:
+ ** *approver* - For the *Qualify* task
+ ** *broker* - For the *Correct Data* and *Increase Down Payment* tasks
+ ** *manager* - For the *Final Approval* task
 
 == Executing a process
 
-Execute the *Mortgage_Process* in {PRODUCT}:
+Execute the *Mortgage Process* process:
 
 . Log in to {PRODUCT}.
-. Click *Menu* -> *Design* -> *Projects*.
-. Click on a project you want to deploy, for example *Mortgage_Process*.
-. Click *Build & Deploy*.
-+
-* If there is no container with the project name, a container with default values is automatically created.
-* A window opens if a container with the project name already exists. You can change the name of your container and its alias. When finished, click *Ok*.
-
-. To verify the deployment, click *Manage > Process Definitions*, and click image:btn_refresh.png[]. Click *Start* to start and instantiate your process.
+. Click *Menu* -> *Manage* -> *Process Definitions*.
+. Click any where in the *MortgageApprovalProcess* row to view the process details.
+. Click the *Diagram* tab to view the business process diagram in the editor.
+. Click on the *X* in the upper-right corner to close the editor.
+. Click any where in the *MortgageApprovalProcess* row and click *New Process Instance* to open the *Application* form.
+. Input the following values in to the form fields:
+
+* *Down Payment*: `30000`
+* *Years of amortization*: `10`
+* *Name*: `Ivo`
+* *Annual Income*: `60000`
+* *SSN*: `123456789`
+* *Age of property*: `8`
+* *Address of property*: `Brno`
+* *Locale*: `Rural`
+* *Property Sale Price*: `50000`
+
+. Click *Submit* to view the *Instance Details*.
+. Click on the *Diagram* tab to view the process flow within the process diagram. The state of the process will be highlighted as it moves through each task.
+. Click *Menu* -> *Track* -> *Task Inbox*. This will take you to the *Qualify* form.
+. As the approver, you will review the *Qaulify* task information, and if approved, select *Is mortgage application in limit?* and click *Complete*.
+. In the *Task Inbox*, click any where in the *Final Approval* row to open the *Final Approval* task.
+. Click *Claim* to claim responsibility for the task, then click *Complete* to finalize the loan approval process.
+
+Note: The *Save* and *Release* buttons are only used to either pause the approval process and save the instance if you are waiting on a field value, or to release the task for another user to modify.
```

Comment:
I mentioned this before in the last review. You need to change all instances of "Note:" to "NOTE:" in order to get the proper rendered format.